tools/dump_kubernetes.sh

```diff
@@ -250,14 +261,16 @@ check_logs_for_errors() {
 }
 
 main() {
+  local exit_code=0
   parse_args "$@"
   check_prerequisites kubectl
   dump_time
   dump_pilot
   dump_resources
-  tap_containers dump_logs_for_container copy_core_dumps_if_istio_proxy
+  if tap_containers dump_logs_for_container copy_core_dumps_if_istio_proxy; then
```

Comment:
The if is still backwards here: `exit_code` is set to 254 if the command succeeds.

I would still recommend removing the if and using `exit_code=tap_containers dump_logs_for_container copy_core_dumps_if_istio_proxy`

Reply:
Done.